corehq/apps/case_importer/tests/test_importer.py

```diff
@@ -436,6 +438,85 @@ def test_columns_and_rows_align(self):
         self.assertEqual(cases['Caroline'].owner_id, case_owner._id)
         self.assertEqual(cases['Caroline'].get_case_property('favorite_color'), 'yellow')
 
+    def test_user_can_access_location(self):
+        with make_business_units(self.domain) as (inc, dsi, dsa), \
+                restrict_user_to_location(self, dsa):
+            res = self.import_mock_file([
+                ['case_id', 'name',             'owner_id'],  # noqa: E241
+                ['',        'Leonard Nimoy',    inc.group_id],  # noqa: E241
```

Comment:
nit: E241 exists so you don't do this either...

Reply:
But, but, but it's a table. ... Hold my beer. I know what to do ...

Reply:
(I [reverted](ffa07744495477d013266f308f9f2a05acfc2215) my [snark](2dd082150b1f16057828d501d03103f0a7635231). Parsing text is fun and all, but after a few hours to think it over it seemed a little silly.)